package.json

```diff
@@ -43,5 +43,8 @@
     "packages/notifi-node-sample",
     "packages/notifi-react-card",
     "packages/notifi-react-hooks"
-  ]
+  ],
+  "dependencies": {
+    "libphonenumber-js": "^1.10.13"
+  }
```

Comment:
just realized this is inside the root package.json

probably want to move this to `notifi-react-card`